gossip/src/cluster_info_metrics.rs

```diff
@@ -434,6 +434,11 @@ pub(crate) fn submit_gossip_stats(
             i64
         ),
         ("push_message_count", stats.push_message_count.clear(), i64),
+        (
+            "num_duplicate_push_messages",
+            crds_stats.num_duplicate_push,
```

Comment:
Generally I would rather use the same name for both the submitted metric and the variable calculating the metric value.
It makes things easier when you observe a metric on chronograf dashboards and want to search the code to figure out how exactly that metric is calculated.

Reply:
ok good point. I want to get this right so lets do that.